cmds/volumes.go

```diff
@@ -49,12 +51,41 @@ func NewCmdVolumes(f *cmdutil.Factory) *cobra.Command {
 				util.Fatal("No default namespace")
 			} else {
 
-				found, pendingClaimNames := findPendingPVS(c, ns)
+				found, pvcs, pendingClaimNames := findPendingPVS(c, ns)
 				if found {
 
 					sshCommand := cmd.Flags().Lookup(sshCommandFlag).Value.String()
 
 					createPV(c, ns, pendingClaimNames, sshCommand)
+					items := pvcs.Items
+					for _, item := range items {
+						pvcName := item.ObjectMeta.Name
+						status := item.Status.Phase
+						if status == "Pending" || status == "Lost" {
+							err = c.PersistentVolumeClaims(ns).Delete(pvcName)
+							if err != nil {
+								util.Infof("Error deleting PVC %s\n", pvcName)
+							} else {
+								util.Infof("Recreating PVC %s\n", pvcName)
+								strs := []string{ns, pvcName}
+								c.PersistentVolumeClaims(ns).Create(&api.PersistentVolumeClaim{
+									ObjectMeta: api.ObjectMeta{
+										Name:      pvcName,
+										Namespace: ns,
+									},
+									Spec: api.PersistentVolumeClaimSpec{
+										VolumeName:  strings.Join(strs, "-"),
+										AccessModes: []api.PersistentVolumeAccessMode{api.ReadWriteOnce},
+										Resources: api.ResourceRequirements{
+											Requests: api.ResourceList{
+												api.ResourceName(api.ResourceStorage): resource.MustParse("5Gi"),
```

Comment:
`5Gi` - where does this amount come from? Is it enough? Is it too much? @rawlingsj thoughts?